docs/design/office-ui-fabric.md

```diff
@@ -21,7 +21,7 @@ To get started using Fabric Core:
 1. Add the CDN reference to the HTML on your page.  
```

Comment:
Can you edit to line above - refer to Fabric React only?

Reply:
Good catch, thank you @Reezaali . Done!